src/puppetlabs/trapperkeeper/services/webserver/experimental/jetty9_websockets.clj

```diff
@@ -50,13 +51,16 @@
     (.. this (getSession) (getUpgradeRequest) (isSecure)))
```

Comment:
While we're at it, it'd be nice to stash the remote address as well. We can't log it in on-close because the session goes away.